src/gilded_rose.js

```diff
@@ -21,45 +21,56 @@ const items = [
 
 updateQuality(items);
 */
-export function updateQuality(items) {
+export function updateQuality(itemsArray) {
+  let result = []
+  const items = [...itemsArray]
```

Comment:
I was trying to follow along with this but I wasn't entirely sure what's going on here. What's the reason for using the spread operator with the items array?

For example, let's say I have an array like this:
```
const myFavoriteColors = [ 'green', 'orange' ]
```
If I use the spread operator but also put it inside of an array, like what you're doing on line 26...